dsl_lang/dune-project

```diff
@@ -1,2 +1,29 @@
-(lang dune 3.10)
-(using menhir 2.1)
+(lang dune 3.16)
+
+(name dsl_lang)
+
+(generate_opam_files true)
+
+(source 
+ (github cucapra/packet-scheduling))
+
+(authors
+ "Anshuman Mohan"
+ "Cassandra Sziklai"
+ "Kabir Samsi"
+ "Akash Dhiraj")
+
+(maintainers
+ "Anshuman Mohan"
+ "Cassandra Sziklai"
+ "Kabir Samsi"
+ "Akash Dhiraj")
+
+(using menhir 3.0)
+
+(package
+ (name dsl_lang)
+ (synopsis "A short synopsis")
```

Comment:
Would you like to hit it out of the park and just add a quick couple things here too? Doesn't have to be perfect